frontend/src/modules/lobby/components/InvitationLink.vue

```diff
@@ -48,12 +44,32 @@ export default {
 </script>
 
 <style scoped>
-.invitation-link-section {
-    margin-top: 25px;
+.invitation-link {
+    display: grid;
+    grid-template-columns: auto 300px auto;
+    grid-template-rows: 38px;
+    border-radius: 10px;
+    border: 1px solid black;
 }
 
-#invitation-link {
+.invitation-link__title {
```

Comment:
Super nit-picky, but do any of these actually need to be classes?

Reply:
I hear mostly that the by default you should use classes, and ids should be reserved for special cases, not the other way around.